lambda-http/src/lib.rs

```diff
@@ -1,29 +1,70 @@
 #![warn(missing_docs)]
 //#![deny(warnings)]
-//! Enriches the `lambda_runtime` crate with [http](https://github.com/hyperium/http)
-//! types targeting ALB and API Gateway proxy events.
+//! Enriches the `lambda` crate with [`http`](https://github.com/hyperium/http)
+//! types targeting AWS [ALB](https://docs.aws.amazon.com/elasticloadbalancing/latest/application/introduction.html), [API Gateway](https://docs.aws.amazon.com/apigateway/latest/developerguide/welcome.html) REST and HTTP API lambda integrations.
 //!
-//! Though ALB and API Gateway proxy events are separate Lambda triggers, they both share
-//! similar shapes that contextually map to an http request handler. From a application perspective
-//! the differences shouldn't matter. This crate
-//! abstracts over both using standard [http](https://github.com/hyperium/http) types allowing
-//! you to focus more on your application while giving you to the flexibility to
-//! transparently use whichever http trigger suits your application's needs best.
+//! This crate abstracts over all of these trigger events using standard [`http`](https://github.com/hyperium/http) types minimizing the mental overhead
+//! of the implementation details allowing you to focus more on your application while also giving you to the maximum flexibility to
+//! transparently use whichever lambda trigger suits your application's needs best.
 //!
 //! # Examples
 //!
+//! ## Hello world
+//!
+//! lambda_http handlers adapt to the standard `lambda::Handler` interface using the [`handler`](fn.handler.html) function.
+//!
+//! The simplest case http handler is a closure of `http::Request` to a type that can be lifted into an `http::Response`.
+//! You can learn more about these types [here](trait.IntoResponse.html)
+//!
+//! ```rust,no_run
+//! use lambda_http::{handler, lambda};
+//! type Error = Box<dyn std::error::Error + Send + Sync + 'static>;
+//!
+//! #[tokio::main]
+//! async fn main() -> Result<(), Error> {
+//!     lambda::run(handler(|request| async { Ok("👋 world!") })).await?;
+//!     Ok(())
+//! }
+//! ```
+//!
+//! ## Hello world (simpler)
```

Comment:
```suggestion
//! ## Hello World
```

Let's also show the simple example first?